src/screens/Home/index.tsx

```diff
@@ -1,7 +1,14 @@
 import WidgetGrid from '../../components/WidgetGrid';
+import RideFab from '../../components/RideFab';
+import {View} from 'react-native';
 
 const ActiveRideView = () => {
-  return <WidgetGrid widgetArray={[]} />;
+  return (
+    <View>
+      <RideFab />
```

Comment:
I'm only rendering the Fab button on the Ride page. It was confusing on the other pages with the other buttons.

Reply:
Is this true? weirdly I see it on both the navigation and sensors screens, which looks fine. We probably do need to figure out some way to make it look better over the widget. It kinda looks odd over the `Avg. Speed` widget, but i don't have a good suggestion currently.

Reply:
Ah! Because of the portal it's rendering outside the nav tree. So its not unmounted on route changes. It looks like it has a visibility prop we can toggle. I'll try that